docker/README.md

```diff
@@ -1,24 +1,54 @@
-# Getting Started using Docker
+# Getting Started Using Docker
 
-**Running using docker cli**:
+This guide provides step-by-step instructions on how to set up and run the Docker environment for your GPT-Engineer project.
 
-Building the image:
-- `git clone https://github.com/gpt-engineer-org/gpt-engineer.git`
-- `cd gpt-engineer`
-- `docker build --rm -t gpt-engineer -f docker/Dockerfile .`
+## Prerequisites
 
-Running the container:
-- `docker run -it --rm -e OPENAI_API_KEY="YOUR OPENAI KEY" -v ./your-project:/project gpt-engineer`
+- Docker installed on your machine.
+- Git (for cloning the repository).
 
-The `-v` flag mounts the `your-project` folder into the container. Make sure to have a `prompt` file in there.
+## Setup Instructions
 
-**Running using docker-compose cli**:
+### Using Docker CLI
 
-Building the image:
-- `git clone https://github.com/gpt-engineer-org/gpt-engineer.git`
-- `cd gpt-engineer`
-- `docker-compose -f docker-compose.yml build`
-- `docker-compose run --rm gpt-engineer`
+1. **Clone the Repository**
+   ```
+   git clone https://github.com/gpt-engineer-org/gpt-engineer.git
+   cd gpt-engineer
+   ```
 
+2. **Build the Docker Image**
+   ```
+   docker build --rm -t gpt-engineer -f docker/Dockerfile .
+   ```
 
-Set the OPENAI_API_KEY in docker/docker-compose.yml using .env file or environment variable, and mount your project folder into the container using volumes. for example "./projects/example:/project" ./projects/example is the path to your project folder.
+3. **Run the Docker Container**
+   ```
+   docker run -it --rm -e OPENAI_API_KEY="YOUR_OPENAI_KEY" -v ./your-project:/project gpt-engineer
+   ```
+   Replace `YOUR_OPENAI_KEY` with your actual OpenAI API key. The `-v` flag mounts your local `your-project` directory inside the container. Replace this with your actual project directory.  Ensure this directory contains all necessary files, including the `prompt` file.
+
+### Using Docker Compose
+
+1. **Clone the Repository** (if not already done)
+   ```
+   git clone https://github.com/gpt-engineer-org/gpt-engineer.git
+   cd gpt-engineer
+   ```
+
+2. **Build and Run using Docker Compose**
+   ```
```

Comment:
Here are some alternatives using `docker compose`:
Since there is only one `docker-compose.yml` file, you could run it without the `-f` option.
- `docker compose up -d --build` - Build images before starting containers, when they are built, run the containers in the background
- `docker compose up -d` - Only start the containers in the background
- `docker compose down` - Stops containers and removes containers, and networks
- `docker compose down -v` - Removes the volumes as well

Reply:
Love it!  Please incorporate in the updated README in your forthcoming PR.